.env

```diff
@@ -0,0 +1 @@
+REACT_APP_JWT_SECRET=cbvjsbdfhdsjkfnsthis
```

Comment:
you should not push `.env` to github, remove it

Reply:
yes